docs/conf.py

```diff
@@ -1,79 +1,85 @@
-# -*- coding: utf-8 -*-
+# Configuration file for the Sphinx documentation builder.
 #
-# venusian documentation build configuration file
-#
-# This file is execfile()d with the current directory set to its containing
-# dir.
-#
-# The contents of this file are pickled, so don't put values in the
-# namespace that aren't pickleable (module imports are okay, they're
-# removed automatically).
-#
-# All configuration values have a default value; values that are commented
-# out serve to show the default value.
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
 
-import sys
-import os
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath("."))
 import datetime
 import pkg_resources
+import pylons_sphinx_themes
 
-# Add and use Pylons theme
-if "sphinx-build" in " ".join(sys.argv):  # protect against dumb importers
-    from subprocess import call, Popen, PIPE
-
-    p = Popen("which git", shell=True, stdout=PIPE)
-    git = p.stdout.read().strip()
-    cwd = os.getcwd()
-    _themes = os.path.join(cwd, "_themes")
-
-    if not os.path.isdir(_themes):
-        call(
-            [git, "clone", "git://github.com/Pylons/pylons_sphinx_theme.git", "_themes"]
-        )
-    else:
-        os.chdir(_themes)
-        call([git, "checkout", "master"])
-        call([git, "pull"])
-        os.chdir(cwd)
-
-    sys.path.append(os.path.abspath("_themes"))
-
-    parent = os.path.dirname(os.path.dirname(__file__))
-    sys.path.append(os.path.abspath(parent))
-    wd = os.getcwd()
-    os.chdir(parent)
-    sys.path.append(parent)
 
-# Options for HTML output
-# -----------------------
+# -- Project information -----------------------------------------------------
 
-sys.path.append(os.path.abspath("_themes"))
-html_theme_path = ["_themes"]
-html_theme = "pylons"
-html_theme_options = dict(github_url="https://github.com/Pylons/venusian")
+project = "venusian"
+copyright = "2022, Pylons Project"
+author = "Pylons Project"
 
-# If your extensions are in another directory, add it here. If the
-# directory is relative to the documentation root, use os.path.abspath to
-# make it absolute, like shown here.
-# sys.path.append(os.path.abspath('some/directory'))
+# The full version, including alpha/beta/rc tags
+release = "3.0.0"
```

Comment:
This information should come from the Venusian package, I don't know which project we have it in, but we use the metadata for the package itself to create the version information, so that it is the same as setup.cfg.

Reply:
Actually I was sloppy. Thanks for catching this. I cleaned this up significantly, removing a lot of chaff in https://github.com/Pylons/venusian/pull/75/commits/dd4ff734bef0560846bb35025b9b46a4d825535d